README_ml.md

```diff
@@ -0,0 +1,51 @@
+
+**Background**: 
+  - We store our data in a redshift database, and while we’ve done a lot of good work modeling this raw resource to help ourselves and teams across the organization quickly answer basic questions about fundraising patterns on our site through a BI tool, many of the deeper explorations our team does for the organization starts with manipulating the data stored in our redshift cluster using SQL. 
+  - This query is a messy version of the query we used to pull the data used in exercise two of this take home. All of the where clauses, and table sources are correct and don’t need to be updated, but this is obviously not a great work product to share with colleagues - it’s hard to scan for quick understanding of the work, has inconsistent syntax, and there are some syntactical errors that would cause this query run to fail. 
+
+**Suggested Time**: 30 minutes 
+
+**Notes**:
+  - If you are not familiar with Common Table Expressions (CTEs) check out [this resource](https://learnsql.com/blog/what-is-common-table-expression/)
+  - Everyone has different SQL styles and we are intentionally not sharing our internal style guide, so prioritize consistency over a specific style 
+  - Highly suggested that you leave comments throughout the file to explain your work, express uncertainty, or ask questions
+
+
+### Exercise Two: Model Building and Deployment
+**Objective**: Build a predictive model from ActBlue’s (publicly available) FEC filing
+
+**Materials**:
+A sample of contribution data from ActBlue’s FEC filings:
+  - [`Data_Science_Technical_FEC_Filing_Sample.csv`](https://drive.google.com/file/d/1IO8Tpzip90NwAKdA-VU0soaj2RyNFIk4/view?usp=sharing) is a sample of records derived from 3 of [ActBlue's monthly filings](https://www.fec.gov/data/committee/C00401224/?tab=filings&cycle=2020) in the 2020 cycle: [February Monthly 2020](https://docquery.fec.gov/cgi-bin/forms/C00401224/1385527/), [March Monthly 2020](https://docquery.fec.gov/cgi-bin/forms/C00401224/1391686/), [April Monthly 2020](https://docquery.fec.gov/cgi-bin/forms/C00401224/1402724/).
+  
+FEC committee data from the 2020 cycle:
+  - [`Data Science_Technical_FEC_Committee_Data_2020.csv`](https://drive.google.com/file/d/1qQ-tAEW9VVH-hyafndohCMQ87AKmhL6U/view?usp=sharing) is derived from the Committee Master file for 2019-2020 provided on the [FEC Bulk Data page](https://www.fec.gov/data/browse-data/?tab=bulk-data).
+
+**Suggested Time**: 90 minutes
+
+**Tools**:
+R or Python preferred 
+
+
+**Instructions**:
+
+- Execute and present the data integrity checks you run before analyzing a new data set 
+- Merge the `Data_Science_Technical_FEC_Filing_Sample` data with the `Data Science_Technical_FEC_Committee_Data_2020` data
+- Create a feature set
+- Choose an appropriate predictive model
+- Train the model and evaluate its performance
+- (Optional for all, required for Sr. level) Wrap the model if a very simple API (e.g. Flask, Docker)
```

Comment:
Is this going to be the new technical for all levels? If so, maybe we could bold the Sr.?